linux_os/guide/system/software/integrity/fips/package_dracut-fips_installed/ansible/shared.yml

```diff
@@ -7,4 +7,3 @@
   package:
     name: dracut-fips
     state: present
-  when: ansible_distribution == 'RedHat'
```

Comment:
I think in this case you can get rid of remediations completely. And possibly also get rid of test scenarios. this rule can be partially templated, at least the remediation part, because the only think it does is installing a package.
You can see an example of such rule here:
https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/bootloader-grub2/grub2_kernel_trust_cpu_rng/rule.yml
But in this case, you would use the template "package_installed".

Reply:
I saw in OVAL that it shouldn't be templated, I thought that would also apply to remediations:

> \<!-- DO NOT TEMPLATE this check. dracut-fips should only be installed on 
>  certified systems, and changes to this file should be scrutinized through the
>  review process.
> \-->

https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/software/integrity/fips/package_dracut-fips_installed/oval/shared.xml

Reply:
Hm, I think this does not apply to remediations. I think so because the OVAL is a bit special - it has the line
```
      <extend_definition comment="Installed OS is FIPS certified" definition_ref="installed_OS_is_FIPS_certified" />
```
This line ensures that the rule will be applicable only to systems which fullfill this extend_definition. That's fine. But remediations do not have such special restriction. Therefore, I think remediations can be templated.
A question though: does OL7 have the dracut-fips package?

Reply:
Yes, it has it